searches/double_linear_search.py

```diff
@@ -0,0 +1,59 @@
+"""Implementation of a double-linear-search,
+which iterates through the array from both sides: start and end.
+
+Change ARRAY_LENGTH to a number you want to generate an example array,

```

Comment:
Lines 4 - 10 should be moved into `__main__` because they are not used by your function.